README.md

```diff
@@ -87,4 +97,8 @@ The most common mood is :-)
 
 **think**: Should you use an array or a hash to solve this problem? Why?
 
+I feel like using an array is easier to push the result of analyze_mood into it, then just calculate the length of the arrays.
+
```

Comment:
Interesting. This will definitely work. But it's not a very scalable solution. What if we eventually add a bunch more feelings, like "Afraid :O" or "Silly :P" ... More arrays, and more comparisons, ick...